### DISCUSSION TITLE
automate login flow via github

### DISCUSSION THREAD

**Discussion by gowrisankar22** [concourse/concourse#5762] [General]
Hello,

Our concourse setup is using GitHub based authentication. Currently, for automation, we use the concourse local user(admin user) which works well and good. 

We don't want to use the admin user directly for the automation due to security reasons, rather we want to make use of GitHub credentials to generate the concourse token and use it for the automation. 

Can somebody help, how can I achieve this. ?

Concourse version: 6.2.0

@jwntrs 

BRs, Gowrisankar

**Comment by vito** [comment]
I don't think this is currently possible with Dex: https://github.com/dexidp/dex/issues/1225
